eldritch/mythos.py

```diff
@@ -126,6 +127,37 @@ def json_repr(self, state):
     }
 
 
+def ReturnAndIncrease(monster_names):
```

Comment:
Should these be in events.py? They seem to be common, Mythos-specific patterns, but then so is ReleaseToStreets...